plugins/modules/monitoring/pagerduty_change.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2020, Adam Vaughan (@adamvaughan) avaughan@pagerduty.com
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+module: pagerduty_change
+short_description: Track a code or infrastructure change as a PagerDuty change event
+version_added: 1.3.0
+description:
+  - This module will let you create a PagerDuty change event.
+author:
+  - Adam Vaughan (@adamvaughan)
+requirements:
+  - PagerDuty integration key
+options:
+  integration_key:
+    description:
+      - The integration key that identifies the service the change was made to.
+        This can be found by adding an integration to a service in PagerDuty.
+    required: true
+    type: str
+  summary:
+    description:
+      - A short description of the change that occurred.
+    required: true
+    type: str
+  source:
+    description:
+      - The source of the change event.
+    default: Ansible
+    type: str
+  user:
+    description:
+      - The name of the user or process that triggered this deployment.
+    type: str
+  repo:
+    description:
+      - The URL of the project repository.
+    required: false
+    type: str
+  revision:
+    description:
+      - An identifier of the revision being deployed, typically a number or SHA from a version control system.
+    required: false
+    type: str
+  environment:
+    description:
+      - The environment name, typically C(production), C(staging), etc.
+    required: false
+    type: str
+  link_url:
+    description:
+      - A URL where more information about the deployment can be obtained.
+    required: false
+    type: str
+  link_text:
+    description:
+      - Descriptive text for a URL where more information about the deployment can be obtained.
+    required: false
+    type: str
+  url:
+    description:
+      - URL to submit the change event to.
+    required: false
+    default: https://events.pagerduty.com/v2/change/enqueue
+    type: str
+  validate_certs:
+    description:
+      - If C(no), SSL certificates for the target URL will not be validated.
+        This should only be used on personally controlled sites using self-signed certificates.
+    required: false
+    default: yes
+    type: bool
+notes: Supports C(check_mode).
```

Comment:
`notes` must be a list:
```suggestion
notes:
  - Supports C(check_mode).
```
I would actually change it to
```suggestion
notes:
  - Supports C(check_mode). Note that check mode simply does nothing except returning `changed=true` in case the `url` seems to be correct.
```